### PR TITLE
Update rollup.md to include the updates to interval logic and minimum interval logic

### PR DESCRIPTION
**Pull request by mhintz**
### What does this PR do? What is the motivation?

We've updated the default interval logic for time series graphs, including lowering the minimum intervals substantially. The docs don't yet reflect these changes, so this PR updates them to describe the system more accurately and in a bit more detail

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/dashboards/functions/rollup.md [hunk 1]
```diff
@@ -52,9 +52,9 @@ When graphing, Datadog imposes a limit on the number of points per graph. To res
 | The past day        | 5m                          | 20m                        | 5m                   |
 | The past two days     | 10m                         | 30m                        | 10m                  |
 | The past week       | 1hr                         | 2hr                        | 1hr                  |
-| The past month      | 2hr                         | 12hr                       | 4hr                  |
+| The past month      | 4hr                         | 12hr                       | 4hr                  |
```
Comment (mhintz) [inline content/en/dashboards/functions/rollup.md:55]: Line graphs were aligned with what the API returns for a 1-month time frame

File: content/en/dashboards/functions/rollup.md [hunk 2]
```diff
@@ -43,7 +43,7 @@ Applying the `moving_rollup()` function to a query allows you to combine points
 
 ## Rollup interval: enforced vs custom
 
-When graphing, Datadog imposes a limit on the number of points per graph. To respect this limit, Datadog rolls up data points automatically with the `avg` method, effectively displaying the average of all data points within a time interval for a given metric. This default time interval varies depending on how the data is being visualized. See the following chart to reference these default time intervals:
+When graphing, Datadog sets a limit on the number of points per time series. To retain visual clarity, series can have up to 1500 points. To respect this limit, Datadog rolls up data points automatically, defaulting to the `avg` method, effectively displaying the average of all data points within a time interval for a given metric. The default rollup time interval varies depending on how the data is being visualized. See the following chart to reference these default time intervals:
```
Comment (rtrieu) [inline content/en/dashboards/functions/rollup.md:46]:
```suggestion
When graphing, Datadog sets a limit on the number of points per timeseries. To retain visual clarity, a series can have up to 1500 points. To respect this limit, Datadog rolls up datapoints automatically, defaulting to the `avg` method, effectively displaying the average of all datapoints within a time interval for a given metric. The default rollup time interval varies depending on how the data is visualized. See the following chart to reference these default time intervals:
```

File: content/en/dashboards/functions/rollup.md [hunk 3]
```diff
@@ -52,9 +52,9 @@ When graphing, Datadog imposes a limit on the number of points per graph. To res
 | The past day        | 5m                          | 20m                        | 5m                   |
 | The past two days     | 10m                         | 30m                        | 10m                  |
 | The past week       | 1hr                         | 2hr                        | 1hr                  |
-| The past month      | 2hr                         | 12hr                       | 4hr                  |
+| The past month      | 4hr                         | 12hr                       | 4hr                  |
 
-A custom `.rollup()` function can be used to enforce the type of time aggregation applied (`avg`, `min`, `max`, `count`, or `sum`) and the time interval to rollup.
+A custom `.rollup()` function can be used to enforce the type of time aggregation applied (`avg`, `min`, `max`, `count`, or `sum`) and optionally the time interval to rollup. Using this function, you can set the rollup time interval to a different value than the defaults, up to the limit of 1500 points. This supports up to one point per minute over a day.
```
Comment (rtrieu) [inline content/en/dashboards/functions/rollup.md:57]:
```suggestion
A custom `.rollup()` function can be used to enforce the type of time aggregation applied (`avg`, `min`, `max`, `count`, or `sum`) and optionally the time interval to rollup. Using this function, you can set the rollup time interval to a different value than the defaults, up to a limit of 1500 points. This supports up to one point per minute over a day.
```

**Review by rtrieu — CHANGES_REQUESTED** [review]
Some minor feedback, but otherwise looks good!

**Review by rtrieu — CHANGES_REQUESTED** [review]
Looks good, thanks!

**Review by rtrieu — APPROVED** [review]
Looks good!

**Pull request merged**